### PR TITLE
Fix for issue #350

### PR DESCRIPTION
**Pull request by liz-lawrens**
Fix for issue #350

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,10 +39,12 @@ script:
   - make
   - travis_retry ctest -V
 after_success:
+  if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
   - docker login -u ovckey -p $REGISTRY_PASS
   - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:build 
   - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:latest 
-
+  else
+  fi
```
Comment (1480c1) [inline .travis.yml]:
```suggestion
  - |
    if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
      docker login -u ovckey -p $REGISTRY_PASS
      docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:build 
      docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:latest
    fi

**Comment by jhou5** [comment]
See below error in travis -
/home/travis/.travis/functions: eval: line 110: syntax error: unexpected end of file

is it as expected?

File: .travis.yml [hunk 2]
```diff
@@ -38,11 +38,12 @@ script:
     cd ${CPU}/${buildOS}/${usage}/${imageName}
   - make
   - travis_retry ctest -V
-after_success:
-  - docker login -u ovckey -p $REGISTRY_PASS
-  - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:build 
-  - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:latest 
-
+after_success: |
+   if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```
Comment (1480c1) [inline .travis.yml]: Suggest not using double equal since its not guaranteed with single bracket test or just use double brackets since these run in bash

**Comment by liz-lawrens** [comment]
Wow that was some quick reviews! I was still working on the fix, trying out stuff to get the right syntax. Looks like it works now. Thanks @jhou5 @1480c1 for the quick review 👍

File: .travis.yml (hunk 2 above)
Comment (1480c1) [reply .travis.yml]: Doesn't really matter either way

**Pull request merged**